.gitignore

```diff
@@ -82,6 +86,9 @@ python/_external_repositories/
 
 # created by Dask tests
 python/dask-worker-space
+python/cugraph/dask-worker-space
+python/cugraph/cugraph/dask-worker-space
+python/cugraph/cugraph/tests/dask-worker-space
```

Comment:
These 3 are helpful because I won't have to exclude them myself before pushing a commit